### PR TITLE
Release v1.1.1

### PR DESCRIPTION
**Pull request by jathanism**
Request to merge `develop` into `main` for Release v1.1.1. This is so that we can get the approvals and tests ready to go for the release.

### CODE REVIEW

File: nautobot/docs/release-notes/version-1.1.md [hunk 1]
```diff
@@ -80,14 +80,25 @@ Please see the section on [migrating to Celery from RQ](../installation/services
...
+- [#744](https://github.com/nautobot/nautobot/issues/744) - Fixed missing Celery Django fixup that could cause assorted errors when multiple background tasks were run concurrently.
+- [#759](https://github.com/nautobot/nautobot/issues/759) - Corrected backwards add/import links for Power Feed and Power Panel in navigation bar
 
 ## v1.1.0 (2021-07-20)
 
 ### Added
 
 - [#372](https://github.com/nautobot/nautobot/issues/372) - Added support for displaying custom fields in tables used in object list views
 - [#620](https://github.com/nautobot/nautobot/pull/620) - Config context schemas can now be managed via Git repositories.
+- [#681](https://github.com/nautobot/nautobot/pull/681) - Added an example guide on how to use AWS S3 for hosting static files in production.
```
Comment (glennmatthews) [inline nautobot/docs/release-notes/version-1.1.md:101]: Looks like the release-notes for #681 and #738 (below) are incorrectly added under the 1.1.0 section.

File: pyproject.toml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 name = "nautobot"
 # Primary package version gets set here. This is used for publishing, and once
 # installed, `nautobot.__version__` will have this version number.
-version = "1.1.0"
+version = "1.1.1-beta.1"
```
Comment (glennmatthews) [inline pyproject.toml:5]: Need to update this of course. :-)

Comment (jathanism) [reply pyproject.toml:5]: Yeah! I'll be fixing these all in one shot once #776 is merged!

**Review by glennmatthews — APPROVED** [review]
Approved, pending #776.

**Review by lampwins — APPROVED** [review]

**Comment by glennmatthews** [comment]
@jathanism I took a stab at updating the release-notes this morning - please review when you have a chance. :-)

File: nautobot/docs/release-notes/version-1.1.md [hunk 3]
```diff
@@ -80,7 +74,35 @@ Please see the section on [migrating to Celery from RQ](../installation/services
...
+
+#### HTTP "Remote end closed connection" errors ([#725](https://github.com/nautobot/nautobot/issues/725))
+
+The example `uwsgi.ini` provided in earlier versions of the Nautobot documentation was missing a recommendation to include the configuration `http-keepalive = 1` which enables support for HTTP/1.1 keep-alive headers.
+
+!!! warning
+    If you are upgrading from an earlier version of Nautobot (including 1.1.0) you should check your `uwsgi.ini` and ensure that it contains this important configuration line.
+
+
+## v1.1.1 (2021-??-??)
```
Comment (glennmatthews) [inline nautobot/docs/release-notes/version-1.1.md:87]: Just flagging this as needing the proper date filled in on release day. :-)

**Pull request merged**